_layouts/default.html

```diff
@@ -16,11 +16,13 @@
     <div class="wrapper">
       <header>
         <h1 class="header"><a href="https://buildah.io">Buildah</a></h1>
-        <p class="header">Buildah - a tool that facilitates building OCI images.</p>
+        <p class="header">Buildah - a tool that facilitates building OCI container images.</p>
```

Comment:
This line came from the top of the Buildah GitHub page (https://github.com/containers/buildah).  I don't see a way to change it from my side, can you touch it up there too?